src/data/protocols/recovery-password/token-generator.ts

```diff
@@ -0,0 +1,3 @@
+export interface TokenGenerator {
```

Comment:
It's better rename `Token` to `Code` or `RecoveryCode` or something like this. Because Token naturally remember JWT. At login route task have a JWT implementation with this interface name.

Reply:
I will adopt the interface you created, as I believe it is more generic than mine. And token generation is not specific to only one context.